doc/level1.rst

```diff
@@ -512,7 +534,7 @@ with more than one recipient. These headers MUST be placed in the root
 MIME part of the encrypted message payload. The encrypted payload in
 this case contains one Autocrypt-Gossip header for each recipient,
 which MUST include ``addr`` and ``keydata`` attributes with the relevant
-data from the recipient's Autocrypt peer state.
+data from the recipient's Autocrypt :ref:`peer state <peer-state>`.
```

Comment:
The recipient's AC peer state sounds like it would be taken from the recipient.
Alternative:
'data from the senders Autocrypt peer state about the recipient'